.github/workflows/ci-cd.yml

```diff
@@ -137,7 +137,7 @@ jobs:
   deploy-documentation:
     name: Deploy documentation
     runs-on: ubuntu-latest
-    needs: [lint, test, test-compatibility, test-floating-dependencies, test-node]
+    needs: [lint, test, test-compatibility, test-node]
```

Comment:
The changes in this pull request will help us continuously deploy the documentation site again.